walls.m

```diff
@@ -0,0 +1,210 @@
+(* Created with the Wolfram Language for Students - Personal Use Only : www.wolfram.com *)
+{{-1., -0.9333286285400391, -0.7838773131370544, -0.5647909641265869, 

```

Comment:
These should be calculated fresh each time, I think. Why store them in the repository?

Reply:
These are static for the moment and imported from refined DO files.  I am not calculating them for the moment, but they could be generated each time. I don't know a good way to put in the new points and hit the walls without copying this format for the time being.

Reply:
If you don't want to refine yourself, I think it should be implemented by reading from a "reference" file. This way the data is not duplicated in multiple places.